### PR TITLE
community(bug_report.md): Create a template for bug reports

### PR DESCRIPTION
**Pull request by nharrand**
Hi all,

Following @slarse suggestion in #4220, I suggest adding a template for bug report to encourage people to give us the steps for reproduction.

WDYT?

### CODE REVIEW

**Comment by MartinWitt** [comment]
A small missing detail is the used java version. There are some small differences between some JDK versions, which can cause strange behavior. Otherwise looks good.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+Output:
+```java
+/*Output code or stacktrace if unexpected Exception is raised*/
+```
+
+Note that a pull request containing a test demonstrating the bug is even more welcome than an issue!
+
+**JDK and Spoon version used**
```
Comment (slarse) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Add OS (Windows does funny things) and perhaps provide some more obvious "help"?

```suggestion
**Operating system, JDK and Spoon version used**

* OS:
* JDK:
* Spoon version:
```

**Review by slarse — COMMENTED** [review]
Overall lgtm, had just a couple of comments.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```java
+/*Code calling spoon on the input and triggering the bug.*/
+```
+
+Output:
+```java
+/*Output code or stacktrace if unexpected Exception is raised*/
+```
+
+Note that a pull request containing a test demonstrating the bug is even more welcome than an issue!
```
[the remark this answers is not in the document]
Comment (nharrand) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: That's a good point.

**Review by slarse — APPROVED** [review]
Thanks @nharrand, we should have done this a long time ago :)

**Pull request merged**